### PR TITLE
DF-1531-Refactoring Event related macros

### LINKED ISSUE #295: Refactor and simplify logic in tags macro
See https://github.com/cfpb/cfgov-refresh/pull/288#discussion_r25460334

### PR DESCRIPTION
**Pull request by sebworks**
Refactoring Event related macros

## Changes

- Updates '_event-post-macros.html' file to allow for more flexible date, time, and address formats.
- Updates 'tags.html" file to simplify logic and Fix #295.

## Review

- @jimmynotjim 
- @anselmbradford

### CODE REVIEW

**Comment by anselmbradford** [comment]
See comments in https://github.com/cfpb/cfgov-refresh/pull/313

File: _includes/tags.html [hunk 1]
```diff
@@ -19,19 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}.
 
-   display_header:     The optional boolean flag to display the tags_header element.
-                       Defaults to True. Adds class 'tags__hide-header' if falsy
-                       value passed (e.g., '', False, None).
+   hide_header:        The optional boolean flag to hide the tags_header element.
+                       Defaults to False. Adds class 'tags__hide-header' if True
+                       value passed (e.g., True, True, None).
```
Comment (anselmbradford) [inline _includes/tags.html]: Small typo `(e.g., True, True, None)`

File: events/_event-post-macros.html [hunk 2]
```diff
@@ -104,23 +115,55 @@
...
+                              address information  
+                              (e.g. '{venue} - {city}, {state}').
 
    ========================================================================== #}
 
-{% macro event_meta_address(post) %}
-    {%- set city = (post.location or 'Washington DC') -%}
-    {%- set state = (post.state or '') -%}
-    {%- set venue = (post.venue or 'CFPB') -%}
+{% macro event_meta_address(post, address_format='%city %state') %}
```
Comment (anselmbradford) [inline events/_event-post-macros.html]: Just checking that default value shouldn't be `address_format='{city} {state}' like https://github.com/cfpb/cfgov-refresh/pull/316/files#diff-6bf9522468f4795c50b2040f06dbc410R100

**Comment by anselmbradford** [comment]
See above comments, otherwise looks :+1:

**Comment by sebworks** [comment]
ok thanks @anselmbradford. I want to make some additional changes but I'm not going to do it in this PR. I want to add the capability to a macro which allow us to easily modify elements with BEM modifiers.

**Comment by anselmbradford** [comment]
Cool, sounds good. Yeah if there's a PR you need to get through and there's still minor stuff hanging that isn't breaking the functionality something I'll sometimes do is spin off issues for those comments so they aren't lost and proceed on merging the PR.

**Pull request merged**